python/paddle/fluid/tests/unittests/dygraph_to_static/test_se_resnet.py

```diff
@@ -34,9 +33,14 @@
 PRINT_STEP = 2
 STEP_NUM = 10
 
-place = fluid.CPUPlace()
-# TODO(liym27): Diff exists between dygraph and static graph on CUDA place.
-# place = fluid.CUDAPlace(0) if fluid.is_compiled_with_cuda() else fluid.CPUPlace()
+place = fluid.CUDAPlace(0) if fluid.is_compiled_with_cuda() \
+    else fluid.CPUPlace()
+
+# Note: Set True to eliminate randomness.
+#     1. For one operation, cuDNN has several algorithms,
+#        some algorithm results are non-deterministic, like convolution algorithms.
+if fluid.is_compiled_with_cuda():
+    fluid.core.globals()['FLAGS_cudnn_deterministic'] = True
```

Comment:
If use python api to set flag, recommend to use `set_flags` under `framework.py`

Reply:
Thanks, fixed.